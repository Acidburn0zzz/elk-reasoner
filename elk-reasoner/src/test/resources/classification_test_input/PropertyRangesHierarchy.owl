Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Prefix( rdfs: = <http://www.w3.org/2000/01/rdf-schema#> )
Ontology(<www.example.org#property_ranges_hierarchy>
Annotation(rdfs:label ":R has two range restrictions :B and :C due to super-properties :S and :H. Hence :A has an :S-successor satisfying :B, :C, and :D")

SubObjectPropertyOf(:R :S)
SubObjectPropertyOf(:R :H)
ObjectPropertyRange(:S :B)
ObjectPropertyRange(:H :C)
SubClassOf(:A ObjectSomeValuesFrom(:R :D))
SubClassOf(ObjectSomeValuesFrom(:S ObjectIntersectionOf(:B :C :D)) :E)

)