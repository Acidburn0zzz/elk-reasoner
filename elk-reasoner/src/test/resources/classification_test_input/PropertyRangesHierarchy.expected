Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Ontology(<www.example.org#property_ranges_hierarchy#taxonomy>

SubClassOf(:A :E)
Declaration(Class(:B))
Declaration(Class(:C))
Declaration(Class(:D))

)